tests/resample/conftest.py

```diff
@@ -11,3 +21,71 @@ def wcs_gwcs():
     shape = (1000, 1000)
     pscale = 0.06 / 3600
     return make_gwcs(crpix, crval, pscale, shape)
+
+
+@pytest.fixture(scope="module")
+def nrcb5_wcs_wcsinfo():
+    """ Returns both wcs and wcsinfo """
+    path = Path(__file__).parent.parent / DATADIR / "nrcb5-wcs.asdf"
+    with asdf.open(path, lazy_load=False) as asdf_file:
+        wcs = asdf_file.tree["wcs"]
+        wcs.bounding_box = wcs_bbox_from_shape(wcs.array_shape)
+        wcsinfo = asdf_file.tree["wcsinfo"]
+        return wcs, wcsinfo
+
+
+@pytest.fixture(scope="module")
+def nrcb5_many_fluxes(nrcb5_wcs_wcsinfo):
```

Comment:
Actually, if you don't mind adding photutils as a *test dependency*, you can call this instead:

https://photutils.readthedocs.io/en/latest/user_guide/datasets.html#simulating-images

Reply:
Not worth it (for me):
1. `Gaussian2DKernel` is simple;
2. I want two types of sources: uniform ("extended") and point sources spread across the image;
3. I want a grid so that I can check whether the position in input image affects the resampled flux (distortion effects);
4. I want pixels outside of the patch to be zero so they do not contribute (mix with) border pixels in the patch in order to test _as precise as possible_ flux conservation.

I don't know whether that function can do this simulation